MobileClient/Server/Controllers/CartsController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using DatabaseDefinition.EntityModel.Repositories.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using TM = TransferModel;
+
+namespace MobileClient.Server.Controllers;
+
+public class CartsController : ControllerBase
+{
+    private readonly ICartRepository cartRepository;
+
+    public CartsController(ICartRepository cartRepository)
+    {
+        this.cartRepository = cartRepository;
+    }
+
+    [HttpPost("{userId}")]
```

Comment:
```suggestion
    [HttpPost("{userId}/Items")]
```
Das macht so mehr Sinn, da du in der menge der Carts eins hast und das eine Menge an Items.